content/kb/deployments/authentication-without-sso.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Authentication without SSO
+slug: /knowledge-base/deploy/authentication-without-sso
+---
+
+# Authentication without SSO
+
+## Introduction
+
+Want to secure your Streamlit app with passwords but Streamlit Cloud Teams and Enterprise doesn't support your SSO provider yet? We got you covered! This guide shows you two simple techniques for adding basic authentication to your Streamlit app, using [secrets management](/streamlit-cloud/get-started/deploy-an-app/connect-to-data-sources/secrets-management).
+
+<Warning>
+
+While these techniques were tested and secured on the Streamlit side, they are obviously not comparable to proper authentication with a SSO provider. Please view them as a temporary workaround! For enterprise use-cases, please read [Configuring Single Sign-on (SSO)](/streamlit-cloud/get-started/share-your-app/configuring-single-on-sso).
```

Comment:
I'd remove the "While these techniques were tested and secured on the Streamlit side" for something more generic. Something to the effect of "While this technique adds some level of security, it is NOT comparable to proper authentication..."  I'd also remove "Please view them" sentence.

Reply:
Thank you! I've edited it to read:

> While this technique adds some level of security, it is **NOT** comparable to proper authentication with a SSO provider. For enterprise use-cases, please read [Configuring Single Sign-on (SSO)](/streamlit-cloud/get-started/share-your-app/configuring-single-on-sso).